docs/docs/launch-manual/sections/using-pulsar/sections/pulsar-packages.md

```diff
@@ -119,16 +121,29 @@ $ ppm search linter
 > └── linter-rust Lint Rust-files, using rustc and/or cargo (132550 downloads, 91 stars)
 ```
 
-You can use `ppm view` to see more information about a specific package.
+You can use `pulsar -p view` to see more information about a specific package.
 
 ```sh
-$ ppm view linter
+$ pulsar -p view linter
 > linter
 > ├── 3.4.0
 > ├── https://github.com/steelbrain/linter
 > ├── A Base Linter with Cow Powers
 > ├── 9863242 downloads
 > └── 4757 stars
 >
-> Run `ppm install linter` to install this package.
+> Run `pulsar -p install linter` to install this package.
 ```
+
+#### Using ppm to install from other sources
+
+By default ppm will be using the [Pulsar Package Repository](https://web.pulsar-edit.dev/).
+However you can also use it to install from other locations.
+
+**GitHub**  
+`pulsar -p install <github_username>/<github_project> [-b <branch_or_tag_or_commit>]`
+
+**Git remote**  
+`pulsar -p install <git_remote> [-b <branch_or_tag_or_commit>]`
```

Comment:
I would choose to add a more meaningful example here, like instead of "git_remote" day something like "git URL", and then explain that the URL is the same one uses to clone a repository.

We could also use some package as an example of how this repository thing works, maybe with minimap or generic-lsp

Reply:
Good idea, I'll have a look at it tomorrow.

Reply:
I've reworked this section if you would like to take another look.